### DISCUSSION TITLE
How to reduce text size to make shell look smaller ?

### DISCUSSION THREAD

**Discussion by deepanshpandey** [moudey/Shell#59] [Q&A]
I just wanted to make shell a bit smaller because in my case shell covers a huge vertical space and I want to reduce that if it is possible.

**Comment by moudey** [comment]
Hi @deepanshpandey With these options, you can reduce the size of the context menu
`font` You can make the font smaller or larger, which will change the size of the context menu
`item.margin` and `item.padding` You can modify the values of these two options to adjust the size of the height or width of the items

```
// version 1.8
default
{
    theme
    {
        font.size=12
    
        item
        {

            padding.top=0
            padding.bottom=0

            margin.top=0
            margin.bottom=0
        }
    }
}
```

**Reply by deepanshpandey** [reply]
thank you so much sir
